bleak/backends/bluezdbus/client.py

```diff
@@ -30,11 +30,14 @@
 from bleak.backends.device import BLEDevice
 from bleak.backends.service import BleakGATTServiceCollection
 from bleak.exc import BleakDBusError, BleakError
-
+from .version import BlueZFeatures
 
 logger = logging.getLogger(__name__)
 
 
+_on_ci = "CI" in os.environ
```

Comment:
This should no longer be needed. It was just a hack to prevent import fail so we didn't have to install bluez on CI runs.

Reply:
Removed, re-squashed, and re-tested.  all good 👍